server/davidia/models/messages.py

```diff
@@ -56,6 +56,48 @@ class GlyphType(str, Enum):
     Cap = "Cap"
 
 
+class ColourMap(str, Enum):
+    Blues = "Blues"
```

Comment:
Can define an enum subtype with [_generate_next_value_](https://docs.python.org/3/library/enum.html#enum.Enum._generate_next_value_) and use `auto()`

Reply:
Added `_generate_next_value()` static method and `auto()`.
Could use a list e.g. `["Blues", "Greens", etc]` and `setattr()`  to loop through instead to cut down "Colour = auto()" code.